packages/components/src/components/Alert/Alert.tsx

```diff
@@ -76,6 +75,7 @@ export const Alert: FC<AlertProps> = ({
   testID,
 }) => {
   const colorScheme = useColorScheme()
```

Comment:
```suggestion

```
can remove and update isDarkMode to use `theme`, right?

Reply:
As is, the useTheme() hook only returns an object with the colors, but I could instead update it to return the name as well. e.g.:

```
const { theme, themeName } = useTheme()
```

Reply:
Ah. Probably fine to leave alone then instead of adding complexity to the hook for what should hopefully be rare of needing to alter the component in special ways for light/dark mode.